### DISCUSSION TITLE
Addition of new programs

### DISCUSSION THREAD

**Discussion by PavanKumarCB** [codinasion-archive/.github-re#128] [Ideas]
I was thinking maybe we can add a set of programs to find the roots of a quadratic equation, using the quadratic formula.

**Comment by harshraj8843** [comment]
Hey @PavanKumarCB 

You can submit a new program by creating a new issue with the `Submit Program` issue template 👉🏻 https://github.com/codinasion/program/issues/new?assignees=harshraj8843&labels=add-program&template=submit-program.yml&title=%5BADD+PROGRAM%5D%3A+

For reference, see 👉🏻 https://github.com/codinasion/program/issues/7328
